cloud_functions/token_refresh/main.py

```
@@ -1,314 +1,314 @@
+    # Log all request details at the start
+    structured_logger(
+        message="Request received",
+        method=request.method,
+        headers=dict(request.headers),
+        url=request.url,
+        args=dict(request.args),
+    )
+    try:
+        body = request.get_json(silent=True)
+        structured_logger(message="Request body", body=body)
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)